fog-ovirt.gemspec

```
@@ -16,10 +16,11 @@ Gem::Specification.new do |spec|
   spec.license       = "MIT"
 
   spec.files         = `git ls-files -z`.split("\x0")
-  spec.test_files    = spec.files.grep(%r{^tests\/})
+  spec.test_files    = spec.files.grep(%r{^tests/})
 
   spec.require_paths = ["lib"]
-
+  spec.required_ruby_version = ">= 2.0.0"
+  spec.add_dependency("activesupport")
```

If you need it only for running tests, you can have it as development dependency:
```suggestion
  spec.add_development_dependency("activesupport")
```